src/world/Manager/HousingMgr.h

```diff
@@ -181,7 +181,7 @@ namespace Sapphire::World::Manager
 
     bool hasPermission( Entity::Player& player, Land& land, uint32_t permission );
 
-    void editExterior( Sapphire::Entity::Player& player, uint16_t plot, std::vector< uint16_t > containerList, std::vector< uint8_t > slotList, uint8_t removeFlag );
+    void editAppearance( bool isInterior, Sapphire::Entity::Player& player, const Common::LandIdent& landIdent, std::vector< uint16_t > containerList, std::vector< uint8_t > slotList, uint8_t removeFlag );
```

Comment:
there's no point of this being a reference, it's the size of a ptr and you don't change the original so it's just wasteful